src/application/shell.ts

```diff
@@ -97,7 +97,16 @@ class ApplicationShell extends Widget {
 
     let topPanel = this._topPanel = new Panel();
     let hboxPanel = this._hboxPanel = new BoxPanel();
-    let dockPanel = this._dockPanel = new DockPanel();
+    let dockPanel = this._dockPanel = new DockPanel({
+      renderer: {
+        createTabBar: () => {
+          let bar = new TabBar();
+          bar.addClass('jp-mod-top');
```

Comment:
Why do you need this class here? You know that `p-DockPanel-tabBar` will be at the top, so you can just use that selector.